### PR TITLE
PortGraph Integration

### PR DESCRIPTION
**Pull request by nxanil**
@natabal 

Convert `PortGraph` into the functional component using various HOCs and new library `material-ui/core`. Please have a look.

### CODE REVIEW

File: GraphsV2/PortGraph/index.js [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+        borderRight,
+    } = properties;
+
+    const [portAreaWidth, setStatePortAreaWidth] = useState(100);
+    const [rowCount, setStateRowCount] = useState(0);
+    const [openModal, setOpenModal] = useState(false);
+    const [infoBoxData, setInfoBoxData] = useState({});
+    const [customTooltips, setCustomTooltips] = useState({});
+
+    const setTooltip = () => {
```
Comment (natabal) [inline GraphsV2/PortGraph/index.js]: let's remove unnecessary functions and move useEffect after useState

**Comment by natabal** [comment]
@nxanil Can we have a screenshot? Thanks!

**Comment by nxanil** [comment]
`Port Graph`

[Screenshot from Gyazo](https://gyazo.com/8a64aeb0c6b5c3bc4f2a3c882bc6f022)

**Comment by natabal** [comment]
@nxanil a few weeks ago during out meeting we talked about splitting the port graph to have only the port icons without the the 'columns', can we have this done?

**Comment by nxanil** [comment]
> @nxanil a few weeks ago during out meeting we talked about splitting the port graph to have only the port icons without the the 'columns', can we have this done?



[Screenshot from Gyazo](https://gyazo.com/fc221cbf29920a092f6e7fb7cf8d0be5).

Please have a look.

**Pull request merged**